lib/MediaWords/DBI/Stories/AP.pm

```diff
@@ -52,6 +52,9 @@ use MediaWords::Util::CSV;
 
 use Digest::MD5 qw(md5);
 
+# cache ap media id
+my $_ap_media_id;
```

Comment:
Those global variables that we're using across our code are generally to be avoided because: 1) well, they are global variables (limited to the Perl package scope but accessible from anywhere in principle), not class / instance variables; 2) they're nearly impossible to port to Python -- due to the intricacies of `Inline::Python` they will be reset to `None`. If you'd like to have such a variable, I'd recommend that you consider refactoring the whole thing to a class and keeping `_ap_media_id` as an instance variable (`$self->{ _ap_media_id }` in vanilla Perl classes or `$self->_ap_media_id()` when using Moose).

Reply:
fixed.  just removed the static variable.  this operation already has to do a relatively expensive query of the story_sentences table, so a quick hit to the media tabel is not going to significantly slow it down.